bindings/wasm/bindings.js

```diff
@@ -291,74 +487,85 @@ Module.setup = function() {
     return manifold;
   };
 
-  Module.Manifold.prototype = Object.create(ManifoldCtor.prototype);
+  Module.Manifold.ofMesh = function(mesh) {
+    return new Module.Manifold(mesh);
```

Comment:
Why `ofMesh` instead of just the plain constructor?

Reply:
I feel like `Manifold.ofMesh(m)` is clearer than `new Manifold(m)` (same with the CrossSection constructor alias I added, `ofPolygons`). Also, it stands out as an available option if it is there in the list of functions imo (with a hint of what you need to give it in the name). The regular constructor still exists too.

Reply:
Alright, that sounds reasonable.